cli/src/main.rs

```diff
@@ -88,6 +83,24 @@ async fn sign_up(pool: &PgPool, username_opt: Option<String>) -> color_eyre::Res
     Ok(())
 }
 
+async fn check_status(pool: &PgPool) -> color_eyre::Result<()> {
```

Comment:
Nit: Maybe this could be `check_auth_status`?

Reply:
will do